native/Avalonia.Native/inc/avalonia-native.h

```diff
@@ -308,6 +308,7 @@ AVNCOM(IAvnPlatformThreadingInterface, 0b) : IUnknown
     // Can't pass int* to sharpgentools for some reason
     virtual void Signal(int priority) = 0;
     virtual IUnknown* StartTimer(int priority, int ms, IAvnActionCallback* callback) = 0;
+    virtual HRESULT TerminateApp () = 0;
```

Comment:
Actually it's a bit redundant since we already pass a cancellation to the native run loop here: https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Native/PlatformThreadingInterface.cs#L65

Might as well store it somewhere and call Cancel on that. 

With `TerminateApp` we have two separate APIs for doing the same thing.

Reply:
Done